app/src/main/resources/application.yml

```
@@ -43,7 +43,7 @@ spring:
     oauth2:
       resourceserver:
         jwt:
-          issuer-uri: http://auth-app:9000
+          issuer-uri: http://localhost:9000
```

```suggestion
          issuer-uri: http://localhost:8080
```

`http://auth-app:9000` does not work because the application on this host is running on port 8080 (see prod yml). But if we fix `http://auth-app:8080` here, it will only work in Docker environment. Fits for me. You can change it locally if you start it without docker-compose.